.gitignore

```diff
@@ -8,3 +8,4 @@ apicast/logs/
 *.rock
 lua_modules/
 tmp/benchmark/
+apicast/config/test.lua
```

Comment:
We should allow loading any lua file as a config and make the test runner use a temporary file instead of this one.